src/styles/App.css

```diff
@@ -0,0 +1,68 @@
+header {
+  justify-content: space-between;
+  background-color: var(--white-color);
+  border-bottom: 1px solid var(--lightgrey-color);
+  padding: 20px 100px;
+  margin-bottom: 20px;
+}
+
+.header-leftside > * {
+  margin-right: 60px;
+}
+
+.header-leftside span,
+.header-leftside select {
+  font-size: 15px;
+  border: none;
+  background-color: transparent;
+}
+
+header img {
+  width: 40px;
+  height: 40px;
+}
+
+.BooksList {
+  padding-bottom: 25px;
+  border-bottom: 2px solid var(--lightgrey-color);
+}
+
+.form-container {
+  padding: 40px 100px;
+}
+
+.form-container .title {
+  color: var(--grey-color);
+  font-size: 22px;
+  margin-bottom: 20px;
+}
+
+.BooksForm {
+  justify-content: space-between;
+  width: 100%;
+  margin-bottom: 40px;
+}
+
+.BooksForm > * {
+  height: 40px;
+  padding: 10px;
+  border: 1px solid var(--lightgrey-color);
+  background-color: var(--white-color);
+  border-radius: 3px;
+  font-size: 16px;
+  color: #c4c4c4;
+}
+
+input[type="text"] {
+  width: 60%;
+}
+
+.BooksForm select {
+  width: 20%;
+}
+
+.BooksForm button {
+  width: 15%;
+  background-color: var(--azure-color);
+  color: var(--white-color);
+}
```

Comment:
- [ ] <b>REQUIRED:</b> Your app doesn't look exactly like mockup design

| Original      | Yours |
  | ----------- | ----------- |
| ![80047697-0c3f6280-8506-11ea-9416-a4f1800e2329](https://user-images.githubusercontent.com/45217364/130871998-88863e2f-c823-4b7b-a5c4-20162973ff21.png) | ![scrnli_9_9_2021_1-22-23 AM](https://user-images.githubusercontent.com/45217364/132598367-05da91b8-2703-44f5-93e8-c0bf29061b08.png) |
   - The a circle progress bar missing on each of your book item. 
   -  Around each of your Action buttons, there some dots which are not present in the mockup design, kindly remove them.